administrator/components/com_languages/forms/language.xml

```diff
@@ -117,17 +117,14 @@
 
 	<field
 		name="fallback_lang"
-		type="filelist"
+		type="fallback"
 		label="COM_LANGUAGES_FIELD_FALLBACK_LANG_LABEL"
-		description="COM_LANGUAGES_FIELD_FALLBACK_LANG_DESC"
 		stripext="1"
 		directory="media/mod_languages/images/"
 		hide_none="1"
 		hide_default="1"
 		filter="\.gif$"
```

Comment:
take this off

Reply:
Forget it. I should have looked at your screenshot. :)

Reply:
:)